README.md

```diff
@@ -90,10 +100,18 @@ It can receive the following arguments:
 of your credentials.
 - `aws_secret_access_key` (**REQUIRED**) - an secret access key
  of your credentials.
+- `aws_region` - region name for version 4 signatures. default is
```

Comment:
Thanks, but I'd like to name it `region`.
Because when I named keys like `aws_access_key_id` and `aws_secret_access_key ` , my intention was to mimic the naming style  in  `~/.aws/credential`.

In  `~/.aws/config` , the key `region` is used, so I prefer it .
https://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html